### DISCUSSION TITLE
ΓΕ4 - Μέρος Α - Συνεννόηση για την επιλογή βιογραφίας/μελέτης περίπτωσης

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60-2022/.github#43] [General]
Όπως είθισται, ανοίγω καινούριο thread για να συντονιστούμε μεταξύ μας ώστε να μην ανεβάσουμε τις ίδιες βιογραφίες/μελέτες περίπτωσης. Όποιος βρίσκει τι θέλει να χρησιμοποιήσει, μπορεί να το ποστάρει σε αυτή την συζήτηση προς ενημέρωση των υπολοίπων.

**Comment by atsiano81** [comment]
Unity σαν μελέτη περίπτωσης

**Comment by gregtsag** [comment]
Βιογραφία για τον Morton Heilig

**Comment by sotkot** [comment]
Μελέτη περίπτωσης Sega VR

**Comment by afoscolou** [comment]
Επιλέγω το Super Cockpit ως μελέτη περίπτωσης

**Comment by vkaragia** [comment]
Μελέτη περίπτωσης Virtual Reality Medical Training for the Military

**Comment by dimidimots** [comment]
Καλησπέρα θα γράψω για τον Τom Caudell

**Comment by NikosAndreadakis** [comment]
Θα γράψω για το ενδιάμεσο λογισμικό openXR σαν  μελέτη περίπτωσης.
